specification/sbi_cove.adoc

```diff
@@ -401,6 +425,41 @@ image::tvm_destruction_and_memory_reclamation.svg[]
 [title= "TVM runtime execution"]
 image::tvm_runtime_execution.svg[]
 
+
+
+== COVE Enumeration Extension (EID #0x434F5645 "COVE")
+
+This common CoVE extension enumerates capabilities for CoVE such as number of
```

Comment:
Do we really need an extension for this ? This would be queried by the host only. Correct ?

Reply:
right - but I was thinking this extension may serve for other common enumeration functions in the future - such as enumerating capabilities supported before you start using COVH, COVG, COVI and future extensions.

Reply:
All SBI extensions are optional. The more we add, we have to cover the alternate cases in software which makes life bit complicated if there is no use case at this point.

However, if you have an use case discovering broader supervisor domain capabilities apart from domain IDs, we can define an extension for that. Is that a valid assumption that supervisor software may need to invoke this beyond CC scenario ?

Reply:
that is correct - supervisor domains may be used for other use cases such as creating a TEE/platform security service partition - perhaps we should call this extension "SUPD" and not specific to "COVE"?

Reply:
Yeah. That's what I was thinking.

Reply:
For completeness should we describe it here and move it to a separate (sbi 3.0) spec at the time when other extensions are moved?

Reply:
Yes.

Reply:
@atishp04 ptal